src/aequitas/plotting.py

```diff
@@ -37,31 +36,45 @@ def assemble_ref_groups(disparities_table, ref_group_flag='_ref_group_value',
     """
     ref_groups = {}
     ref_group_cols = \
-        list(disparities_table.columns[disparities_table.columns.str.contains(
+        set(disparities_table.columns[disparities_table.columns.str.contains(
             ref_group_flag)])
 
+    # Note: specific measures is a set
     if specific_measures:
-        ref_group_cols = \
-            [measure + ref_group_flag for measure in specific_measures if
-             measure + ref_group_flag in ref_group_cols]
+        specific_measures.intersection({label_score_ref})
```

Comment:
is this meant to update `specific_measures`? i do not believe this is an in-place (by reference) operation. (there is however `intersection_update` or `&=`.)

Reply:
used .union() instead, and made into an assignment